README.md

```diff
@@ -61,15 +60,18 @@ npm install --save-dev i18next-parser@next
 [Gulp](http://gulpjs.com/) defines itself as the streaming build system. Put simply, it is like Grunt, but performant and elegant.
 
 ```javascript
-const i18nextParser = require('i18next-parser').gulp;
-
-gulp.task('i18next', function() {
-  gulp.src('app/**')
-    .pipe(new i18nextParser({
-      locales: ['en', 'de'],
-      output: 'locales/$LOCALE/$NAMESPACE.json'
-    }))
-    .pipe(gulp.dest('./'));
+const i18nextParser = require("i18next-parser").gulp;
+
+gulp.task("i18next", function() {
+  gulp
+    .src("app/**")
+    .pipe(
+      new i18nextParser({
+        locales: ["en", "de"],
```

Comment:
If you want to adopt Prettier for formatting, it can format markdown as well as the contents of code blocks within markdown according to repo rules.

Reply:
Ah, I tried to just use vscode's default code formatting, and it shouldn't change the actual code, like adding semicolons. Probably prettier's formatting slipped through, I'll go through the code and check.

Reply:
In general though, I like linting and prettier, and it would be healthy to add it to the repo imo.

Reply:
As I mentioned, you are welcome to do that. I'm just worried about having consistency throughout the project so that any contributor can follow the same guideline. Feel free to open a PR that prettifies everything!